api/server/server_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/docker/docker/api/server/router"
```

Comment:
delete this line to fix dep problem @denismakogon

Reply:
thanks, haven't seen this line in a diff, my bad =/